bootstrap/fabric/build.gradle.kts

```
@@ -88,9 +88,9 @@ tasks {
     remapJar {
         dependsOn(shadowJar)
         inputFile.set(shadowJar.get().archiveFile)
-        archiveBaseName.set("Geyser-Fabric")
+        archiveBaseName.set("geyser-fabric")
+        archiveVersion.set(project.version.toString() + "+build."  + System.getenv("GITHUB_RUN_NUMBER"))
         archiveClassifier.set("")
-        archiveVersion.set("")
     }
 }
 
```

I would just add a new task like this and remove all other changes. Then just set `remapModrinthJar` as the task for modrinth.
```suggestion

    remapModrinthJar {
        dependsOn(shadowJar)
        inputFile.set(shadowJar.get().archiveFile)
        archiveBaseName.set("geyser-fabric")
        archiveVersion.set(project.version.toString() + "+build."  + System.getenv("GITHUB_RUN_NUMBER"))
        archiveClassifier.set("")
    }
```

---

> I would just add a new task like this and remove all other changes. Then just set `remapModrinthJar` as the task for modrinth.

does it work? i thought you could only use actual loom tasks.

---

Apologies... you would have to import the remap jar task and then register this with that type

```kt
import net.fabricmc.loom.task.RemapJarTask

//...

tasks {
    //...

    register("remapModrinthJar", RemapJarTask::class) {
        dependsOn(shadowJar)
        inputFile.set(shadowJar.get().archiveFile)
        archiveBaseName.set("geyser-fabric")
        archiveVersion.set(project.version.toString() + "+build."  + System.getenv("GITHUB_RUN_NUMBER"))
        archiveClassifier.set("")
    }
}

//...

modrinth {
//...

uploadFile.set(tasks.getByPath("remapModrinthJar"))

//...
}
```